caas/ecs/constants/constants.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package constants
```

Comment:
To avoid importing `ecs` package when we need those constants from outside (workers for example).
If we can ensure no need to import these `StorageProviderType` etc constants later, then we can move those constants to `ecs` package.